README.md

```diff
@@ -98,7 +120,11 @@ By default, sensor has measurement capabilities (for wifi and bluetooth). Every
 - [sensor] wraps collected information into `measurementContent` and sends `measurementContent` on `measurement/mySensorSimId/measurementType` topic.
 - [maestro] receives message, check `mySensorSimId` in DB, creates Sensor if needed, and then creates Measurement in DB.
 
-## Command sending sequence
+### Command sending sequence
+
+You can send [commands](https://github.com/anthill/pheromon/api/clients/Admin/ReadMe.md) to the sensors.
+
+**Sequence**
 
 - [some admin client] sends a `command` to `[mySensorSimId]` through socketIO.
 - [maestro] receives `command` from client, and forwards it to the corresponding sensors through MQTT.
```

Comment:
The sequence end when the sensor send back the result of the command on `/cmdResult/mySensorSimId`